src/middleware/validateRequest.ts

```diff
@@ -5,11 +5,16 @@ export const validateRequest =
   (schema: { body?: Schema; query?: Schema }) =>
   (req: Request, res: Response, next: NextFunction) => {
     if (schema.query) {
-      const { error } = schema.query.validate(req.query);
+      const { error, value } = schema.query.validate(req.query);
+
       if (error) {
         res.status(400).send({ error: error.details[0].message });
         return;
       }
+
+      // As`joi` does not mutate original object with default values
+      // we needed those defaults to be used in controller
+      req.query = value;
```

Comment:
what do we need the value for?
You should not update the codes when writing tests.

Reply:
I will expand on the comment: As`joi` does not mutate original object with default values we needed those defaults to be used in controller.

There are default values defined for the query parameters in the schema, which are not been populated by `joi` and returned as a new object, which we need to assign back to `req.query` as the controller are dependent on it for the desired behavior.